src/services/completions.ts

```diff
@@ -183,6 +185,20 @@ namespace ts.Completions {
         return { isGlobalCompletion: false, isMemberCompletion: false, isNewIdentifierLocation: false, entries };
     }
 
+    function specificKeywordCompletionInfo(keywords: readonly SyntaxKind[]): CompletionInfo {

```

Comment:
Making a way to return specific keywords is extra credit beyond just fixing the obvious regression; the behavior in 4.2 was not great:

<img width="561" alt="image" src="https://user-images.githubusercontent.com/3277153/118540802-d15a2f00-b705-11eb-8581-8709376135f6.png">